contrib/syslog-debun

```diff
@@ -345,18 +376,28 @@ acquire_general_info () {
 acquire_syslog_config () {
 	echo "Copy configs from $confdir"
 	cd $confdir
-	mkdir  ${tmpdir}/config
-	find . | cpio -pd ${tmpdir}/config
+	mkdir ${tmpdir}/config
+	find . > ${tmpdir}/syslog.etc.files
+	grep '\.key$' ${tmpdir}/syslog.etc.files > ${tmpdir}/syslog.etc.files.removed
+	grep -v '\.key$' ${tmpdir}/syslog.etc.files | \
+		while read FILE; do \
+			if ${grepq} "PRIVATE KEY-----" "${FILE}" 2>/dev/null; then
+				echo "${FILE}" >> ${tmpdir}/syslog.etc.files.removed
+			else
+				echo "${FILE}"
+			fi
```

Comment:
I wouldn't drop out any file, even if it's name end with .key., or if the contain the "PRIVATE KEY --- "string.
I have some foggy memories about I started to investigate this area, but I'm not sure, if I gave you notes about that, but...
Don't throw out keys unconditionally!
I suggest to check if openssl is exits, end openssl rsa -modulus or sg. can print out the modulus of a private key.
So, if you keep the certs, their modulus can be printed to, and you can check if a keyfile and a cert file belongs together without knowing the secret key or broke customer's privacy.
One more thing: I don't know if sng (or the underlying openssl) is able to handle cert and key in the same file, but, it would be wise, if you don't throw out the whole file, which contains private key, just find out the line number where the private key starts, where it ends, cut out that part, and put back a placeholder into what debug bundle creator collects, and that placeholder can contain the modulus.
Be careful! I tried that, and I know it's not easy!
You will bump into another nice thing, that head and tail are not exactly follows posix conventions on linuces, because they are gnu. eg. if you want to print a file from the 11.th line, then tail -n +11 may work on gnu versions, and as far as I remember, tail +11 is needed on non-gnu unices. However, that's another thing what can be detected, and it's better to detect that based on error codes and behaviour! ;-)